<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParksDisplay</class>
 <widget class="QDialog" name="ParksDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>491</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nameButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sort by State</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Visitors for Original List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>140</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Acres for Original List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="parkNameButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sort by Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="areaButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sort by Area</string>
   </property>
  </widget>
  <widget class="QPushButton" name="visitorsButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sort by Visitors</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="updatedVisitorsLineEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>210</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="updatedAcresLineEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>270</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Updated Visitors:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>250</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Updated Acres:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="originalVisitorsLineEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>330</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="originalAcresLineEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>380</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>310</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Original Visitors:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>360</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Original Acres:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
